### DISCUSSION TITLE
Error listing display adapters: wrong PCI location property value... Headers, possibly?

### DISCUSSION THREAD

**Discussion by sociofall** [terminatorul/NvStrapsReBar#11] [Q&A] [answered]
Hi there. Could i get your help too?
[HOME-PC.txt](https://github.com/terminatorul/NvStrapsReBar/files/14227799/HOME-PC.txt)
[LocalPciGPU.h.txt](https://github.com/terminatorul/NvStrapsReBar/files/14227806/LocalPciGPU.h.txt)
Extracted data looks correct to me. Firmware also flashed succesfully. Also, my motherboard have built-in ReBar support. Should I disable it?

GPU-Z:
![изображение](https://github.com/terminatorul/NvStrapsReBar/assets/67294769/23df2408-8057-48e2-bec0-f67bbbdbdb2f)

**Comment by terminatorul** [comment, marked as answer]
This happened when the selected language in Windows is not English.

Can you build the latest version from the `bleed` branch in the repository ? That fixed the issue and does not depend on language. Or you could manually fix the regexp in DeviceList.cc

**Reply by sociofall** [reply]
redone everything from scratch, but using bleed branch. seems enabled now, thank you! :)
![изображение](https://github.com/terminatorul/NvStrapsReBar/assets/67294769/054dbbb8-accd-4482-b21c-cfb641ca5993)
